plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -131,7 +137,11 @@ func (l *persistentVolumeLabel) getEBSVolumes() (aws.Volumes, error) {
 	defer l.mutex.Unlock()
 
 	if l.ebsVolumes == nil {
-		cloudProvider, err := cloudprovider.GetCloudProvider("aws", nil)
+		var cloudConfigReader io.Reader
+		if len(l.cloudConfig) > 0 {
+			cloudConfigReader = bytes.NewReader(l.cloudConfig)
+		}
+		cloudProvider, err := cloudprovider.GetCloudProvider("aws", cloudConfigReader)
```

Comment:
Is it possible to process the config file outside of the plugin, and pass in the cloudProvider object instead?  I'm not sure if there's a function to get the current cloud provider?